app/src/pages/movies/movie/Movie.jsx

```diff
@@ -0,0 +1,97 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { EditMovieForm } from './components/EditMovieForm'
+import { DeleteMovieForm } from './components/DeleteMovieForm'
+import { MovieDescription } from './components/MovieDescription'
+import { MovieComponent } from './components/layouts/MovieComponent'
+import { EditDeleteOptions } from './components/EditDeleteOptions'
+
+class Movie extends React.Component {
+    state = {
+        editDeleteShown: false,
+        editMovieFormOpened: false,
+        deleteMovieFormOpened: false
+    }
+
+    showEditDelete = () => {
+        this.setState({
+            editDeleteShown: true
+        });
+    }
+
+    hideEditDelete = () => {
+        this.setState({
+            editDeleteShown: false
+        });
+    }
+
+    openEditMovieForm = () => {
+        this.setState({
+            editMovieFormOpened: true
+        });
+    }
+
+    closeEditMovieForm = () => {
+        this.setState({
+            editMovieFormOpened: false
+        });
+    }
+
+    openDeleteMovieForm = () => {
+        this.setState({
+            deleteMovieFormOpened: true
+        });
+    }
+
+    closeDeleteMovieForm = () => {
+        this.setState({
+            deleteMovieFormOpened: false
+        });
+    }
+
+    render() {
+        return (
+            <>
+                <EditMovieForm 
+                    opened={this.state.editMovieFormOpened} 
+                    close={this.closeEditMovieForm} 
+                    movie={this.props.movie}
+                />
+                <DeleteMovieForm

```

Comment:
`{this.state.deleteMovieFormOpened && (<DeleteMovieForm close={this.closeDeleteMovieForm} />)}`

Reply:
Fixed